rust-connectors/models/fluvio-model-postgres/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "fluvio-model-postgres"
```

Comment:
You've mentioned that the `fluvio-model-postgres` create will be used in smartmodules, let's add a github action to ensure that this (and future) event model crates run `cargo check -p ${{matrix.model-crate}} --target wasm32-unknown-unknown`